packages/web/tests/components/UserProfile/UserProfile.test.tsx

```diff
@@ -16,24 +17,43 @@ getMock(useRouter).mockReturnValue({
 } as any);
 
 const sampleUser: UserProfileProps['user'] = {
+  id: '123',
   name: 'Steve Job',
   pronouns: 'he/him',
   schoolName: 'Apple University',
 };
 
+jest.mock('../../../src/components/userprofile/EditUserForm');
+getMock(EditUserForm).mockImplementation(({ ...EditFormProps }) => <div>This is a form</div>);
+
 describe('Mock UserProfileLayout component', () => {
```

Comment:
Not sure how this snuck through as the describe, but we should never have `Mock` in the description because we shouldn't be testing a mock component...

Should this be `describe('UserProfile component'...` ?